### ISSUE TITLE
Supply a delegate function to indicate that Rollout.io async loading if finished

### ISSUE THREAD

**Issue by ekeren** [rollout/rollout.io-ios#4]

**Comment by KiranPanesar** [comment]
:+1: Would love to see this or a block-based solution like so:

```objectivec
[[RolloutSDK sharedManager] loadFixesWithCompletionHandler:(void(^)(id someParameter)) { 
    // Called when the fixes are loaded.
}];

**Comment by silyevsk** [comment]
Hi @KiranPanesar,

Since the callback (or delegate method or notification) might be called (or sent) in different situations, I'd like to ask you which of the following options would fit your request:
1) once - after the configuration is fetched from network for the first time
2) once - after the configuration is applied for the first time (it's different from option 1 because (starting 0.9) fetched configuration is cached and is loaded from local storage during [Rollout setup] at the next app start
3) every time new configuration is fetched from network (in addition to first setup, fetching happens every time the app enters foreground)
4) every time the configuration is applied (like option 3 and also after applying the cached configuration at the app start)

Also, it would be great if you could can explain the specific use case (or cases) you have in mind.

Thanks,
Sergey.

**Comment by ekeren** [comment]
@KiranPanesar  - any updates?

**Comment by Hunter-Dolan** [comment]
+1 on this.

Where I really see this useful is to be run after Rollout is activated and is at the point where it can do it's job.

So in response to @silyevsk: Once every time the app is launched, after the configuration from local storage is applied and anything new has been fetched over network and applied. So basically on app launch (but not resuming from background), after Rollout has completely loaded for that session.

Basically where I can see this being useful is if you have a piece of code that runs on startup that you may want to disable later without pushing an update. It would be considerably easier to make a hot patch than to write something that hits your server.
